docs/jwst/superbias/reference_files.rst

```diff
@@ -1,23 +1,58 @@
-Reference File Types
-====================
-The superbias subtraction step uses a SUPERBIAS reference file.
+Reference Files
+===============
+The ``superbias`` subtraction step uses a SUPERBIAS reference file.
 
-.. include:: ../includes/standard_keywords.rst
+SUPERBIAS Reference File
+------------------------
+
+:REFTYPE: SUPERBIAS
+:Data model: `~jwst.datamodels.SuperBiasModel`
+
+The SUPERBIAS reference file contains a 2-D image of the detector bias
+("zeroth" read) structure.
 
 .. include:: superbias_selection.rst
 
-SUPERBIAS Reference File Format
--------------------------------
-Superbias reference files are FITS files with 3 IMAGE extensions and 1 BINTABLE
-extension. The FITS primary data array is assumed to be empty. The 
-characteristics of the three image extension are as follows:
+.. include:: ../includes/standard_keywords.rst
+
+Type Specific Keywords for SUPERBIAS
+++++++++++++++++++++++++++++++++++++
+In addition to the standard reference file keywords listed above,
```

Comment:
All of the superbias updates look good to me. Extremely minor comment: the 'standard reference file keywords' are in the rst file in the /includes directory, so would it be clearer to have 'the standard reference file keywords included above...' ?

Reply:
When you're reading the built version of the docs (html, pdf, ...) it looks as if the standard keywords are listed directly within the document being read. It's only under the hood that we use the include mechanism for efficiency. So I think the wording is OK as is.